moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -44,6 +44,16 @@
 #include <boost/bind.hpp>
 #include <moveit/robot_model/aabb.h>
 
+namespace moveit
+{
+namespace core
+{
+/** \brief At least 10 steps per trajectory for testing jump thresholds with computeCartesianPath. Otherwise
+ * testJointSpaceJump doesn't work well. */
+static const std::size_t MIN_STEPS_FOR_JUMP_THRESH = 10;
+}
+}
```

Comment:
Aside: I think I met some resistance in the past to this idea, but I'd rather wrap the whole header with the namespace to improve code readability.